devops/ansible/roles/large_image/tasks/mapnik.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Install gdal system package
+  apt:
+    name:
+      - gdal-bin
+      - libgdal-dev
+      - libmapnik-dev
+      - python-gdal
+      - python-mapnik
+  become: true
+  become_user: root
+
+- name: Install gdal python bindings
+  environment:
+    CPLUS_INCLUDE_PATH: /usr/include/gdal
+    C_INCLUDE_PATH: /usr/include/gdal
+  pip:
+    name: gdal
+    virtualenv: "{{ large_image_virtualenv }}"
+    version: 1.11.2
```

Comment:
Can you collapse these two steps down to put the logic for computing `version` into a single step? Then all of the other step parameters will be shared.